jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/milestones/CaseMilestoneItemView.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<div class="list-group-item">
+    <div class="list-view-pf-main-info">
+        <div class="list-view-pf-body">
+            <div class="list-view-pf-description">
+                <div class="list-group-item-heading"><span data-field="milestone-identifier"></span></div>
+                <div class="list-group-item-text"><span data-field="milestone-name"></span></div>
+            </div>
+        </div>
+        <div class="list-view-pf-left">
```

Comment:
Prefer to use _list-view-pf-actions_ after _list-group-item_ as indicated from PatternFly: https://www.patternfly.org/pattern-library/content-views/list-view/#/_code